Firmware/planner.h

```diff
@@ -233,7 +235,7 @@ FORCE_INLINE block_t *plan_get_current_block()
     return(NULL); 
   }
   block_t *block = &block_buffer[block_buffer_tail];
-  block->busy = true;
```

Comment:
Note this is called in the Stepper ISR. Is it safe to drop the `volatile` keyword?

Reply:
I worry about race conditions here

Reply:
I think yes. I didn't see any place where volatile was really needed.

Reply:
Added @wavexx as reviewer, I think he would know best if this is OK.